.vib/grafana-loki/cypress/cypress/integration/grafana_loki_spec.js

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+import body from '../fixtures/logs-body.json';
+
+it('shows build info endpoint', () => {
+  cy.request({
+    method: 'GET',
+    url: '/loki/api/v1/status/buildinfo',
+    form: true,
+  }).as('buildInfo');
+  cy.get('@buildInfo').should((response) => {
```

Comment:
There is nothing wrong with this notation (aliasing), but conceptually they were introduced to access properties yielded by an async request at a later moment in time. Check: https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Requests

When we are no performing any other operation in between, I personally consider the use of `then()` way neater than aliasing:

```suggestion
  }).then((response) => {
```

Reply:
Applied the said suggestion.